backend/users/services/contributorsStatsController.js

```diff
@@ -0,0 +1,107 @@
+import { promises as fsPromises } from "fs";
+import path from "path";
+
+// fetch contributors from file
+const fetchContributorsFromFile = async () => {
+  try {
+    const filename = path.join("services", "./../../.all-contributorsrc");
+    const filedata = await fsPromises.readFile(filename, "utf8");
+
+    const data = JSON.parse(filedata);
```

Comment:
i think adding error handling here to check if parsing is successful would be helpful